tests/suites/test_suite_pk.function

```diff
@@ -469,7 +469,7 @@ void pk_rsa_decrypt_test_vec( data_t * cipher, int mod, int radix_P,
                               int radix_N, char * input_N, int radix_E,
                               char * input_E, data_t * clear, int ret )
 {
-    unsigned char output[1000];
+    unsigned char output[256];
```

Comment:
How are we guaranteeing that 256 is enough?

Reply:
I ran the tests on PC and printed the values.  This is the maximal size